mne/viz/topomap.py

```diff
@@ -167,6 +167,7 @@ def _average_fnirs_overlaps(info, ch_type, sphere):
                 # first listed channel is the one to be replaced with merge
                 overlapping_set = [chs[i]['ch_name'] for i in
                                    np.where(overlapping_mask[chan_idx])[0]]
+                overlapping_set = np.array(overlapping_set).astype('>U20')
```

Comment:
I would even use lists here to avoid issues cause 20 might be too small too for some people

Reply:
agree that the better fix is to keep `overlapping_set` as a list until *after* the insertion (`list.insert()` ought to work right?), and only then cast to array (if casting to array is even necessary?)

Reply:
Agree with this. I had been thinking the same thing, but had aimed to keep things as close as possible to original code. 

Have updated the PR, now it aggregates a list in the loop, and converts to an array after the loop is finished.